### ISSUE TITLE
Pack and Upload a Solution Error

### ISSUE THREAD

**Issue by mlnc21** [megel/powerapps-helper#20]
PAH Version: 1.8.1
PAC Version: 1.17.5+gcbb58e8

When I try to use the Function “Pack and Upload a Solution”, I become the current Error Message.
![image](https://user-images.githubusercontent.com/80674540/187372811-5ba2a9dc-5eb8-4acc-9460-40454ddc1695.png)
But when I use “Pack Solution” and Manually Upload, then it works.

**Comment by megel** [comment]
Hi @mlnc21 thanks for reporting. I'll check this.

**Comment by mlnc21** [comment]
Hi @megel,
Thanks for the Response.
The size is 8310 KB.

**Comment by megel** [comment]
Found this related to your reported error:
https://github.com/axios/axios/issues/4888#issuecomment-1206848584 

I'll add this to the upload parameters:
```
await axios.post(url, data,{
                headers: headers,
                maxBodyLength: 104857600, //100mb
                maxContentLength: 104857600 //100mb
})
```

**Comment by mlnc21** [comment]
Vielen Dank, Herr Megel  :-)

Beste Grüße aus Bautzen ;)

**Comment by megel** [comment]
Gern geschehen Martin! Der fix ist in v1.8.2 ... bitte testen.
